.changeset/long-boxes-joke.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
we add changeset for packages only. please delete this file since your changes don't require one.

Reply:
I was unsure whether to do so, but in your Contribution.md it's described as:

> "If you made minor changes like CI config, prettier, etc., you can run pnpm changeset add --empty to generate an empty changeset file to document your changes."

Overall, this changeset shouldn't affect versioning at all.